### PR TITLE
chore: added general requirements for 3.4

### PR DESCRIPTION
**Pull request by oldgiova**
Signed-off-by: Roberto Giovanardi <roberto.giovanardi@northern.tech>

# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [x] Make sure that all commits follow the conventional commit [specification](https://www.github.com/mendersoftware/mendertesting/commitlint/grammar.md) for the Mender project.

The majority of our contributions are fixes, which means your commit should have
the form below:

```
fix: added general requirements page

Changelog: Added general requirements page
Ticket: ME-7
```

- [x] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Added general requirements page with AWS S3 policy and with URL allowlist

Thank you!

### CODE REVIEW

File: 02.Overview/16.Requirements/docs.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: Requirements
+taxonomy:
+    category: docs
+---
+
+## Working with Firewalls
+Corporate Firewalls must permit the following URLs to the devices and for letting Mender works correctly:
```
Comment (oleorhagen) [inline 02.Overview/16.Requirements/docs.md:8]:
```suggestion
Corporate Firewalls must permit the following URLs to the devices in order for Mender to work correctly:
```

**Review by oleorhagen — APPROVED** [review]

**Review by alfrunes — APPROVED** [review]

File: 02.Overview/16.Requirements/docs.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+title: Requirements
+taxonomy:
+    category: docs
+---
+
+## Working with Firewalls
+Corporate Firewalls must permit the following URLs to the devices in order for Mender to work correctly:
+
+**Mender access**: Devices, APIs and browser access
+* `https://mender.example.com` - It's the same address you specify it in the [Server Installation Section](../../07.Server-installation/)
```
Comment (TheYoctoJester) [inline 02.Overview/16.Requirements/docs.md:11]:
```suggestion
* `https://mender.example.com` - Use the same address you specify it in the [Server Installation Section](../../07.Server-installation/)
```

File: 02.Overview/16.Requirements/docs.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+**Artifact storage access**: Devices, APIs and browser access
+
+[ui-tabs position="top-left" active="0" theme="default" ]
+[ui-tab title="Amazon S3"]
+If you use Amazon S3 for storing artifacts, you have to permit access to the following URLs:
```
Comment (TheYoctoJester) [inline 02.Overview/16.Requirements/docs.md:17]:
```suggestion
Permit access to the following URLs:
```

Comment (oldgiova) [reply 02.Overview/16.Requirements/docs.md]: Done

File: 02.Overview/16.Requirements/docs.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+[ui-tab title="Minio"]
+If you use Minio for storing artifacts, you have to permit access to your 
+Minio URL defined in [Minio Setup Section](../../07.Server-installation/04.Production-installation-with-kubernetes/04.Minio/docs.md)
+* `https://artifacts.example.com`
+[/ui-tab]
+[/ui-tabs]
+
+
+## Amazon S3 IAM policies
+If you are using Amazon S3 bucket for storing Mender Artifacts, you can use the following minimum policy set:
```
Comment (TheYoctoJester) [inline 02.Overview/16.Requirements/docs.md:30]:
```suggestion
A minimum policy set to use an Amazon S3 bucket to store Mender Artifacts is:
```

**Review by TheYoctoJester — CHANGES_REQUESTED** [review]

**Comment by lluiscampos** [comment]
@oldgiova Could you please sqash all changes into a single commit?

File: 02.Overview/16.Requirements/docs.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+---
+title: Requirements
+taxonomy:
+    category: docs
+---
+
+## Working with Firewalls
+Corporate Firewalls must permit the following URLs to the devices in order for Mender to work correctly:
```
Comment (TheYoctoJester) [inline 02.Overview/16.Requirements/docs.md:8]:
```suggestion
The following URLs and access types need outgoing permissions in firewalls in order for Hosted Mender to work correctly:
```

(copied over from #1913 review)

**Review by TheYoctoJester — CHANGES_REQUESTED** [review]
Please use the sentence on firewalls in the suggestion, then I'm good with it.

File: 02.Overview/16.Requirements/docs.md (hunk 5 above)
Comment (oldgiova) [reply 02.Overview/16.Requirements/docs.md]: Done

**Review by TheYoctoJester — APPROVED** [review]

**Comment by oldgiova** [comment]
@mender-test-bot cherry-pick to:
3.4.x
3.3.x

**Comment by oldgiova** [comment]
@mender-test-bot cherry-pick to:
* 3.4.x
* 3.3.x

**Pull request merged**